sql/xsql/src/main/scala/org/apache/spark/sql/xsql/XSQLSessionCatalog.scala

```diff
@@ -80,6 +80,8 @@ private[xsql] class XSQLSessionCatalog(
   val xsqlExternalCatalog =
     externalCatalogWithListener.unwrapped.asInstanceOf[XSQLExternalCatalog]
 
+  setCurrentDatabase(getCurrentCatalogDatabase.get.name)
+
```

Comment:
I find there exists duplicate call of `getCurrentCatalogDatabase`. Could you avoid it?

Reply:
I think there should add some comments about the reason.

Reply:
I haven’t thought of a better method so far, do you have any suggestions？

Reply:
```
  {
    val catalogDB = getCurrentCatalogDatabase.get
    setCurrentDatabase(catalogDB.dataSourceName, catalogDB.name)
  }
```
I think this will be better.